PluginShowcase/.gitignore

```diff
@@ -0,0 +1,337 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Why a new .gitignore?

Reply:
dotnet avalonia.mvvm does this.

Reply:
I can remove it if we want, but in general I don't fight with tools much. :-)

Reply:
I'll remove them all and see if anything new shows up in status.